go.mod

```diff
@@ -73,35 +73,35 @@ require (
 	github.com/syndtr/goleveldb v1.0.1-0.20210305035536-64b5b1c73954
 	github.com/tidwall/gjson v1.9.1
 	github.com/tidwall/sjson v1.2.2
-	github.com/tikv/client-go/v2 v2.0.0-rc.0.20220107040026-d22815099720
-	github.com/tikv/pd v1.1.0-beta.0.20220207063535-9268bed87199
+	github.com/tikv/client-go/v2 v2.0.0-rc.0.20220217140116-589cf79b2ab5
+	github.com/tikv/pd v1.1.0-beta.0.20220303060546-3695d8164800
+	github.com/tikv/pd/client v0.0.0-20220216080339-1b8f82378ee7
 	github.com/tinylib/msgp v1.1.0
 	github.com/uber-go/atomic v1.4.0
 	github.com/vmihailenco/msgpack/v5 v5.3.5
 	github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
 	github.com/xitongsys/parquet-go v1.6.0 // indirect
-	go.etcd.io/etcd v0.5.0-alpha.5.0.20210512015243-d19fbe541bf9
+	go.etcd.io/etcd/api/v3 v3.5.2
+	go.etcd.io/etcd/client/pkg/v3 v3.5.2
+	go.etcd.io/etcd/client/v3 v3.5.2
+	go.etcd.io/etcd/pkg/v3 v3.5.2
+	go.etcd.io/etcd/server/v3 v3.5.2
+	go.etcd.io/etcd/tests/v3 v3.5.2
 	go.uber.org/atomic v1.9.0
 	go.uber.org/goleak v1.1.12
 	go.uber.org/multierr v1.7.0
-	go.uber.org/zap v1.19.1
+	go.uber.org/zap v1.20.0
 	golang.org/x/net v0.0.0-20211020060615-d418f374d309
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	golang.org/x/sys v0.0.0-20211216021012-1d35b9e2eb4e
 	golang.org/x/text v0.3.7
 	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba
 	google.golang.org/genproto v0.0.0-20210825212027-de86158e7fda
-	google.golang.org/grpc v1.40.0
+	google.golang.org/grpc v1.43.0
 	gopkg.in/mgo.v2 v2.0.0-20190816093944-a6b53ec6cb22 // indirect
 	gopkg.in/yaml.v2 v2.4.0
-	sigs.k8s.io/yaml v1.2.0 // indirect
 	upper.io/db.v3 v3.7.1+incompatible
 )
 
-replace (
-	// Fix CVE-2020-26160.
-	github.com/dgrijalva/jwt-go v3.2.0+incompatible => github.com/golang-jwt/jwt v3.2.1+incompatible
-	// cloud.google.com/go/storage will upgrade grpc to v1.40.0
-	// we need keep the replacement until go.etcd.io supports the higher version of grpc.
-	google.golang.org/grpc v1.40.0 => google.golang.org/grpc v1.29.1
```

Comment:
Why we need to pin grpc to `v1.29.1` before this PR

Reply:
introduced in https://github.com/pingcap/tiflow/pull/2833 @maxshuang PTAL

Reply:
grpc has directory changes, old etcd cannot use newer grpc.